extensions/skyportal/static/js/components/TNSInfo.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+
+import * as tnsInfoActions from "../ducks/tnsInfo";
+
+const TNSInfo = ({ objID }) => {
+  const dispatch = useDispatch();
+  const [requestSubmitted, setRequestSubmitted] = useState(false);
+  const tnsInfo = useSelector((state) => state.tnsInfo);
+
+  useEffect(
+    function fetchTNSInfo() {
+      if (!requestSubmitted) {
+        dispatch(tnsInfoActions.fetchTNSInfo(objID));
+        setRequestSubmitted(true);
+      }
+    },
+    [objID, dispatch, requestSubmitted]
+  );
+
+  if (tnsInfo === null) {
+    return <>Fetching TNS data...</>;
+  }
+  return (
+    <span>
+      {tnsInfo?.name ? (
+        <a
+          href={`https://www.wis-tns.org/object/${tnsInfo.name}`}
+          target="_blank"
```

Comment:
```suggestion
          target="_blank"
          rel="noopener noreferrer"
```

otherwise browsers will complain - see https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/.

Reply:
done